internal/cluster/cluster.go

```diff
@@ -59,15 +73,20 @@ func NewCluster() (Cluster, error) {
 	return cluster, nil
 }
 
-func (c *Cluster) Name() string {
-	// Handle not named clusters
-	for _, prop := range c.Cib.Configuration.CrmConfig.ClusterProperties {
-		if prop.Id == clusterNameProperty {
-			return prop.Value
-		}
+func getCorosyncAuthkey(corosyncKeyPath string) (string, error) {
```

Comment:
Maybe we should rename the method to `getCorosyncAuthkeyMd5`

Reply:
Agreed